pkg/pgclient/config_test.go

```diff
@@ -0,0 +1,182 @@
+package pgclient
+
+import "testing"
+
+func TestConfig_GetConnectionStr(t *testing.T) {
+	type fields struct {
+		Host                    string
+		Port                    int
+		User                    string
+		Password                string
+		Database                string
+		SslMode                 string
+		DbConnectRetries        int
+		AsyncAcks               bool
+		ReportInterval          int
+		LabelsCacheSize         uint64
+		MetricsCacheSize        uint64
+		SeriesCacheSize         uint64
+		WriteConnectionsPerProc int
+		MaxConnections          int
+		UsesHA                  bool
+		DbUri                   string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		want    string
+		wantErr bool
+		err     error
+	}{
+		{
+			name: "test1",
```

Comment:
Love the new tests, would like to see them named a bit better so we know what the test case it is. That way, it's easier to see what we are testing and is the test setup properly.